node/chainSimulator/chainSimulator_test.go

```diff
@@ -404,3 +370,42 @@ func computeTxHash(chainSimulator ChainSimulator, tx *transaction.Transaction) (
 	txHasBytes := chainSimulator.GetNodeHandler(1).GetCoreComponents().Hasher().Compute(string(txBytes))
 	return hex.EncodeToString(txHasBytes), nil
 }
+
+func sendTxAndGenerateBlockTilTxIsExecuted(t *testing.T, chainSimulator ChainSimulator, tx *transaction.Transaction) {
+	shardID := chainSimulator.GetNodeHandler(0).GetShardCoordinator().ComputeId(tx.SndAddr)
+	err := chainSimulator.GetNodeHandler(shardID).GetFacadeHandler().ValidateTransaction(tx)
+	require.Nil(t, err)
+
+	txHash, err := computeTxHash(chainSimulator, tx)
+	require.Nil(t, err)
+	log.Warn("send transaction", "txHash", txHash)
```

Comment:
Why warn?

Reply:
Added this WARN to see when the transaction was sent and check after how many blocks the transaction was executed.